packages/trpc/server/organisation-router/update-organisation.ts

```diff
@@ -47,4 +48,12 @@ export const updateOrganisationRoute = authenticatedProcedure
         url: data.url,
       },
     });
+
+    if (updatedOrganisation.customerId) {
+      await stripe.customers.update(updatedOrganisation.customerId, {
+        metadata: {
+          organisationName: data.name,
```

Comment:
I did `metadata: generateStripeOrganisationCreateMetadata(data.name, userId)` previously, but it creates an object named `organisationCreateData` as follows:

```
organisationCreateData
{"organisationName":"dummy name","userId":0}
```